src/scripts/modules/media/body/body.coffee

```diff
@@ -1,11 +1,24 @@
 define (require) ->
   $ = require('jquery')
+  embeddablesConfig = require('cs!configs/embeddables')
   Mathjax = require('mathjax')
   router = require('cs!router')
   EditableView = require('cs!helpers/backbone/views/editable')
   template = require('hbs!./body-template')
   require('less!./body')
 
+  embeddableTemplates = {
+    'exercise' : require('hbs!./embeddables/exercise-template'),
+    'iframe' : require('hbs!./embeddables/iframe-template')
+  }
```

Comment:
As a general style we've been dropping multi-line brackets and commas when possible:
```coffeescript
embeddableTemplates =
  'exercise' : require('hbs!./embeddables/exercise-template')
  'iframe' : require('hbs!./embeddables/iframe-template')
```